controller.js

```diff
@@ -11,4 +11,55 @@ let ping = async (req, res) => {
 
```

Comment:
In this file we shouldn't be creating a new Express instance. That should only be done in the root file, which passes its Express object into the routes module which appends controllers to specific routes to handle requests. Those handles should be unit tested.

Reply:
Noted. Will make amendments in the next push. Thanks for the feedback